src/test/java/com/epam/tc/hw1/SubOperationTest.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tc.hw1.CalculatorTesting;
+import com.epam.tc.hw1.data.TestData;
+import org.testng.annotations.Test;
+
+
+public class SubOperationTest {
+
+    @Test(dataProvider = "testData", dataProviderClass = TestData.class, groups = "first")
+    public void subTest(long a, long b, long sum, long sub, long multiply, long div) {
+        assertThat(CalculatorTesting.initCalc().sub(a, b)).as("Ups, calculation was wrong").isEqualTo(sub);
```

Comment:
Why do you invoke `CalculatorTesting.initCalc()` instead of extending `SubOperationTest` from `CalculatorTesting`?